analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SerializationConstructorsShouldBeSecured_Part1.cs

```diff
@@ -6,7 +6,7 @@
 namespace MyLibrary
 {
 
-    // Noncompliant - in Part2.cs
+
```

Comment:
What are the empty lines for?

Reply:
That was an old trick to overcome the fact that issues were not checked in the correct files